addons/mission/functions/fnc_ping.sqf

```diff
@@ -0,0 +1,63 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Mike
+ * Creates a sonar-like marker ping on a specific location.
+ * The ping will remain at the location it started on.
+ * List of available colours can be found at: https://community.bistudio.com/wiki/Arma_3:_CfgMarkerColors
+ *
+ * Only run on the server.
+ *
+ * Arguments:
+ * 0: Location <OBJECT>
+ * 1: Marker Name (must be unique unless tracking the same object) <STRING>
+ * 2: Max size of ping <NUMBER> (default: 60, max 120)
+ * 3: Colour <STRING> (default: "ColorRed")
+ *
+ * Return Value:
+ * None
+ *
+ * Examples:
+ * [MyObject, "UniqueName"] call MFUNC(ping)
+ */
+
+params ["_location", "_markerName", ["_maxSize", 60], ["_colour", "ColorRed"]];
+
+if (!isServer) exitWith {
+    [QGVAR(ping), [_location, _markerName, _maxSize, _colour]] call CBA_fnc_serverEvent;
```

Comment:
Since we are not adding an ACE action here and instead documenting it, maybe the documentation of this function should require call on server or directly call `CBA_fnc_serverEvent`.

Reply:
That's a good idea, I'll write up the documentation with an ACE action example which only calls the serverEvent.